lewis/adapters/stream.py

```diff
@@ -86,7 +84,44 @@ def handle_accept(self):
             StreamHandler(sock, self.target)
 
 
-class Cmd(object):
+class CanNotProcessException(Exception):
+    pass
+
+
+class BaseCommand(object):
+    member = None
+    argument_mappings = None
+    return_mapping = None
+    doc = None
+
+    def process_request(self, request, target):
+        raise NotImplementedError('Commands must implement process_request method.')
+
+    @property
+    def patterns(self):
+        raise NotImplementedError('Commands must implement patterns property.')
+
+    def map_arguments(self, arguments):
+        """
+        Returns the mapped function arguments. If no mapping functions are defined, the arguments
+        are returned as they were supplied.
+
+        :param arguments: List of arguments for bound function as strings.
+        :return: Mapped arguments.
+        """
+        if self.argument_mappings is None:
+            return arguments
+
+        return [f(a) for f, a in zip(self.argument_mappings, arguments)]
+
+    def map_return_value(self, return_value):
+        if self.return_mapping is None:
+            return return_value
+
+        return self.return_mapping(return_value)
```

Comment:
I don't know if this will make Codacy happy (to be fair, it has a point), but you could do this instead:
```python
    def map_return_value(self, return_value):
        if callable(self.return_mapping):
            return self.return_mapping(return_value)

        return return_value
```
Or even:
```python
    def map_return_value(self, return_value):
        if callable(self.return_mapping):
            return self.return_mapping(return_value)
        if self.return_mapping is not None:
            return self.return_mapping

        return return_value
```
This would have the side-effect of allowing unconditional, static return overrides in addition to lambda's and other callables.